code/modules/unit_tests/breath.dm

```diff
@@ -1,46 +1,67 @@
+/// Allocates a new Human, and equips them with the given tank type and a breathing tube.
+/datum/unit_test/breath_sanity/proc/equip_labrat(mob/living/carbon/human/lab_rat, tank_type)
+	var/obj/item/clothing/mask/breath/tube = allocate(/obj/item/clothing/mask/breath)
+	var/obj/item/tank/internals/source = allocate(tank_type)
+	lab_rat.equip_to_slot_if_possible(tube, ITEM_SLOT_MASK)
+	lab_rat.equip_to_slot_if_possible(source, ITEM_SLOT_HANDS)
+	return source
+
 /// Tests to make sure humans can breath in normal situations
 /// Built to prevent regression on an issue surrounding QUANTIZE() and BREATH_VOLUME
 /// See the comment on BREATH_VOLUME for more details
-/datum/unit_test/breath_sanity
```

Comment:
Bring this back--we have it here for documentation (you are currently documenting /Run(), not the unit test) and because it makes it easier to `TEST_FOCUS`.

Reply:
Oops